README.md

```diff
@@ -30,25 +30,25 @@ To learn more about graphviz please have a look at http://www.graphviz.org/.
 
 ``` bash
 # install dependencies
-npm install
+yarn install
 
 # serve with hot reload at localhost:8080
-npm run dev
+yarn run dev
 
 # build for production with minification
 npm run build
```

Comment:
Thanks for your changes.

And you're right I'm using yarn because there is a `yarn.lock` file.
Please also modify the build command to `yarn build`.
The `run` is not needed for yarn.

After this everything looks good.